src/components/CommitHistory/HistoryMain/History.css

```diff
@@ -0,0 +1,109 @@
+.HistoryMain {
+  overflow: hidden;
+  height: 100%;
+  width: 100%;
+  display: grid;
+  grid-template-columns: 30% 60% 10%;
+}
+
+.codeview {
+  display: grid;
+  grid-template-rows: 15% 70% 15%;
+}
+
+.Code-Match {
+  background-color: #242a3c;
+  text-align: left;
+  padding-top: 34px;
+  padding-left: 10px;
+}
+
+.LargeButton {
+  align-content: right;
+  color: #ffffff;
+  width: 241px;
+  height: 48px;
+  font-weight: 600;
+  font-size: 24px;
+  font-style: normal;
+  background-color: #3894b1;
+  border: 0px;
+  border-radius: 2px;
+}
+
+.SmallButton {
+  align-content: right;
+  color: #ffffff;
+  width: 80px;
+  height: 48px;
+  background: #1a3051;
+  font-weight: 600;
+  font-size: 24px;
+  font-style: normal;
+  border: 0px;
+  border-radius: 2px;
+}
+
+.LargeButton:hover,
+.SmallButton:hover {
```

Comment:
Due to some reason, this animation is visible only when you toggle to code. But when one toggles to Map, it is not visible. fix it.